### ISSUE TITLE
Way to disable logs on live.

### ISSUE THREAD

**Issue by drinchev** [Adyen/adyen-web-sdk-sample-code#24]
Hello 

I would like to ask if there is a way to disable the logs in `live` context in the live WebSDK. 

```
### checkoutMain::setFocus:: holderDiv=  checkoutSDK.1.9.2.min.js:15
...
```

I don't want the logs to pollute the console of my users

**Comment by Trigalti** [comment]
Hey @drinchev , we will release a new version of our SDK to resolve this issue., as this is an unintended log. I will keep you updated.

**Comment by drinchev** [comment]
Hey @Trigalti thanks for the attention. Btw do you have an ETA for that?

**Comment by pabloai** [comment]
Hi @drinchev 
The fix will be included in version `1.9.10` which should be in live the 5th of August.
